gymnasium/envs/registration.py

```diff
@@ -543,31 +587,21 @@ def make(
                     f"{e}. Environment registration via importing a module failed. "
                     f"Check whether '{module}' contains env registration and can be imported."
                 )
-        spec_ = registry.get(id)
 
         ns, name, version = parse_env_id(id)
-        latest_version = find_highest_version(ns, name)
-        if (
-            version is not None
-            and latest_version is not None
-            and latest_version > version
-        ):
-            logger.warn(
-                f"The environment {id} is out of date. You should consider "
-                f"upgrading to version `v{latest_version}`."
-            )
-        if version is None and latest_version is not None:
-            version = latest_version
-            new_env_id = get_env_id(ns, name, version)
-            spec_ = registry.get(new_env_id)
-            logger.warn(
-                f"Using the latest versioned environment `{new_env_id}` "
-                f"instead of the unversioned environment `{id}`."
-            )
+        spec_ = _search_spec(id, ns, name, version, registry)
 
         if spec_ is None:
-            _check_version_exists(ns, name, version)
-            raise error.Error(f"No registered env with id: {id}")
+            spec_ = _search_in_gym(id, ns, name, version)
+
+            if spec_ is not None:
+                disable_env_checker = True
```

Comment:
This should be `spec_.disable_env_checker=True`. 
Or we modify the passive environment checker such that it only checks that the observation and action spaces exist not the type isinstance Gymnasium.Space which I believe is the current issue.